mm2src/portfolio/portfolio.rs

```diff
@@ -27,54 +26,52 @@
 #[macro_use] extern crate unwrap;
 
 pub mod prices;
-use self::prices::{lp_btcprice, lp_fundvalue, Coins, CoinId, ExternalPrices, FundvalueRes, PricingProvider, PriceUnit};
+use self::prices::{lp_btcprice, lp_fundvalue, CoinId, Coins, ExternalPrices, FundvalueRes, PriceUnit, PricingProvider};
```

Comment:
Could you please roll back the changes unrelated to the actual work? I think we will remove the portfolio crate entirely as it's too far behind the actual code state.

Reply:
Done